src/constants/services.js

```diff
@@ -0,0 +1,15 @@
+import adoptImage from "../assets/image/services/Group 46.png";
```

Comment:
The images in the `assets` are still not named the same as the import - in shortcut, they are still `Group 46.png` and not `adoptImage.png` for example